pkg/common/providers/rosaprovider/vpc.go

```diff
@@ -0,0 +1,119 @@
+package rosaprovider
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/terraform-exec/tfexec"
+	"github.com/openshift/osde2e/assets"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/terraform"
+)
+
+type HyperShiftVPC struct {
+	PrivateSubnet     string
+	PublicSubnet      string
+	NodePrivateSubnet string
+}
+
+// copyFile copies the srcFile provided to the destFile
+func copyFile(srcFile string, destFile string) error {
+	srcReader, err := assets.FS.Open(srcFile)
+	if err != nil {
+		log.Printf("error opening %s file: %s", srcFile, err)
+		return err
+	}
+	defer srcReader.Close()
+
+	destReader, err := os.Create(destFile)
+	if err != nil {
+		log.Printf("error creating runtime %s file: %s", destFile, err)
+		return err
+	}
+	defer destReader.Close()
+
+	_, err = io.Copy(destReader, srcReader)
+	if err != nil {
+		log.Printf("error copying source file to destination file: %s", err)
+		return err
```

Comment:
wrap these errors instead of logging them

Reply:
Resolved in https://github.com/openshift/osde2e/pull/1558/commits/0de34542c4d705ab0513096a30d520fbf86d25bb